.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on: [push]
+
+env:
+  COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+  GRADLE_OPTS: -Xmx3200m -XX:+HeapDumpOnOutOfMemoryError -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy="in-process" -Dorg.gradle.daemon=false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Should be `[self-hosted, AWS, Linux]` I do believe, otherwise this is going to run on Github's runners

Reply:
Missed this - I tried that first, but it wasn't able to match a run. I presumed that was internal-only, but I'll put up a new PR and test it out - maybe they scaled down over the weekend and were saturated?

Reply:
d'oh, right, forgot this is public. Ignore me! :)